src/app/team/page.tsx

```diff
@@ -0,0 +1,301 @@
+"use client";
+
+import { useEffect } from "react";
+import { useAppDispatch, useAppSelector, useTranslations } from "hooks";
+import {
+  teamSlice,
+  ActivityStatus,
+  Expertise,
+  Contributor,
+  selectFilteredContributors,
+} from "state/teamSlice";
+import { fetchTeamState } from "state/teamSlice";
+import { DexterButton } from "components/DexterButton";
+import { FaTelegram, FaGithub } from "react-icons/fa";
+import { SkeletonRectangle } from "components/Skeleton";
+
+export default function Team() {
+  const dispatch = useAppDispatch();
+  useEffect(() => {
+    dispatch(fetchTeamState());
+  }, [dispatch]);
+
+  return (
+    <div className="bg-[#141414] grow flex items-center justify-center pt-10">
```

Comment:
Can we use dexter-grey-dark?
found in https://github.com/DeXter-on-Radix/website/blob/main/tailwind.config.js

Reply:
fixed!